.project

```diff
@@ -14,4 +14,15 @@
 	<natures>
 		<nature>org.eclipse.m2e.core.maven2Nature</nature>
 	</natures>
+	<filteredResources>
```

Comment:
I wonder if this file (and such config files) can be added in the gitignore list? No need to keep committing into them. Thanks

Reply:
Done

Reply:
I think these should stay in per this description I got from [our .gitignore generator](https://www.toptal.com/developers/gitignore) for Eclipse:

# Uncomment this line if you wish to ignore the project description file.
# Typically, this file would be tracked if it contains build/dependency configurations:
#.project